apis/server/container_bridge.go

```diff
@@ -224,17 +224,17 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
 
 func (s *Server) getContainer(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
 	name := mux.Vars(req)["name"]
-	containerInfo, err := s.ContainerMgr.Get(name)
+	meta, err := s.ContainerMgr.Get(ctx, name)
 	if err != nil {
 		return err
 	}
 
 	container := types.ContainerJSON{
-		ID:      containerInfo.ID,
-		Name:    containerInfo.Name,
-		Image:   containerInfo.Config.Image,
-		Created: containerInfo.State.StartedAt,
-		State:   &types.ContainerState{Pid: containerInfo.State.Pid},
+		ID:      meta.ID,
+		Name:    meta.Name,
+		Image:   meta.Config.Image,
+		Created: meta.State.StartedAt,
+		State:   &types.ContainerState{Pid: meta.State.Pid},
```

Comment:
We should add more data here for `ContainerState`.
Currently when we inspect a container, in `State`, there is only one PID, without any other state details.
Maybe in a follow-up.

Reply:
Yes, do it in the follow-up pr.